content/zh/docs/faq/applications/reuse-the-same-app-name-after-deletion.md

```diff
@@ -6,43 +6,43 @@ linkTitle: "删除应用后复用相同应用名称"
 Weight: 16920
 ---
 
-To deploy an app in KubeSphere, tenants can go to the App Store and select the available app based on their needs. However, tenants could experience errors when deploying an app with the same app name as that of the deleted one. This tutorial demonstrates how to use the same app name after its deletion.
+若要在 KubeSphere 中部署应用，租户可以进入应用商店，根据自己的需求选择可用的应用。但是，租户在部署与被删除的应用名称相同的应用时，可能会遇到错误。本教程演示了如何在删除应用后复用相同应用名称。
 
-## Prerequisites
+## 准备工作
 
-- You need to use an account invited to your project with the role of `operator`. This tutorial uses the account `project-regular` for demonstration purposes. For more information, refer to [Create Workspaces, Projects, Accounts and Roles](../../../quick-start/create-workspace-and-project/).
-- You need to [enable the App Store](../../../pluggable-components/app-store/).
+- 您需要使用被邀请到项目中、且具有 `operator` 角色的帐户。本教程使用 `project-regular` 帐户进行演示。有关更多信息，请参考[创建企业空间、项目、帐户和角色](applewebdata://0F652EF3-B63C-436A-8364-B1E1935575CA/quick-start/create-workspace-and-project/)。
+- 您需要[启用应用商店](applewebdata://0F652EF3-B63C-436A-8364-B1E1935575CA/pluggable-components/app-store/)。
 
-## Reuse the Same App Name
+## 复用相同应用名称
 
-### Deploy an app from the App Store
+### 从应用商店中部署应用
 
-1. Log in to the web console of KubeSphere as `project-regular` and deploy an app from the App Store. This tutorial uses Redis as an example app and set the app name as `redis-1`. For more information about how to deploy Redis, refer to [Deploy Redis on KubeSphere](../../../application-store/built-in-apps/redis-app/).
+1. 以 `project-regular` 身份登录 KubeSphere 的 Web 控制台，从应用商店中部署应用。本教程使用 Redis 作为示例应用，将应用名称设置为 `redis-1`。有关更多如何部署 Redis 的信息，请参考[在 KubeSphere 上部署 Redis](applewebdata://0F652EF3-B63C-436A-8364-B1E1935575CA/application-store/built-in-apps/redis-app/)。
 
-   ![redis-1](/images/docs/faq/applications/use-the-same-app-name-after-deletion/redis-1.PNG)
+   ![redis-1](/images/docs/zh-cn/faq/reuse-the-same-app-name-after-deletion/redis-1.png)
```

Comment:
The image should be saved in the directory `/images/docs/zh-cn/faq/applications/use-the-same-app-name-after-deletion`. You can create the `applications` folder manually.